pom.xml

```diff
@@ -322,6 +322,7 @@ flexible messaging model and an intuitive client API.</description>
     <extra-enforcer-rules.version>1.6.1</extra-enforcer-rules.version>
     <oshi.version>6.4.0</oshi.version>
     <checkerframework.version>3.33.0</checkerframework.version>
+    <it-unimi-dsi-fastutil.version>8.5.15</it-unimi-dsi-fastutil.version>
```

Comment:
there's already `fastutil.version` property for this purpose.

Reply:
fixed.